README.md

```diff
@@ -59,15 +59,28 @@ Depending on the platform on which the application is deployed, a different set
 If a feature missing, additional help is needed, or you need to extend the package,
 please reach out to us for support.
 
-<img src="./doc/features.svg">
+| Feature                                              | Android, Android TV, Fire TV            | Web                                                                        | iOS, tvOS                                                                                                             |
+|:-----------------------------------------------------|:----------------------------------------|:---------------------------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------|
+| **Streaming**                                        | >                                       | MPEG-DASH (fmp4, CMAF), HLS (TS, CMAF), Progressive MP4, MP3               | HLS (TS, CMAF), Progressive MP4, MP3                                                                                  |
```

Comment:
A lone `>` inside a table cell is rendered by our documentation website as "merge with cell to the right", i.e. the cell to the right becomes `<td colspan="2">`:
![image](https://github.com/THEOplayer/react-native-theoplayer/assets/649348/99dc0ff5-da77-4894-bd08-a819117c162a)

It does look a little bit odd on GitHub itself, though. See [README.md](https://github.com/THEOplayer/react-native-theoplayer/blob/be9cc078156dc16539cdc0f1d5dfc79601b10e74/README.md#features):
![image](https://github.com/THEOplayer/react-native-theoplayer/assets/649348/461b81e7-7b3a-45e6-81fc-1ef3bbbbf0f7)

Thoughts?

Reply:
Never mind, I switched to a plain HTML `<table>`. That works everywhere. 😛

![image](https://github.com/THEOplayer/react-native-theoplayer/assets/649348/6266705b-4077-466d-bd60-1f5dc7c629b4)